src/main/resources/validationrules/iac/iac-case-role-validation-admin.drl

```diff
@@ -0,0 +1,68 @@
+import uk.gov.hmcts.reform.roleassignment.domain.model.enums.RoleCategory;
+import uk.gov.hmcts.reform.roleassignment.domain.model.enums.RoleType
+import uk.gov.hmcts.reform.roleassignment.domain.model.ExistingRoleAssignment;
+import uk.gov.hmcts.reform.roleassignment.domain.model.CaseAllocatorApproval;
+import uk.gov.hmcts.reform.roleassignment.domain.model.Case;
+import function uk.gov.hmcts.reform.roleassignment.domain.service.common.ValidationModelService.logMsg;
+import uk.gov.hmcts.reform.roleassignment.domain.model.FeatureFlag;
+import uk.gov.hmcts.reform.roleassignment.domain.model.enums.FeatureFlagEnum;
+
+/*
+ * The below rules are required for Specific and Challenged access
+ * It will be toggled on with new IAC_SPECIFIC_1_0 and IAC_CHALLENGED_1_0 DB flag.
+ */
+
+/*
+ * IAC "specific-access-admin" case creation and case-allocator as a assigner
+ */
+rule "case_allocator_create_specific_access_admin_case_role"
+when
+    $f : FeatureFlag(status && flagName == FeatureFlagEnum.IAC_SPECIFIC_1_0.getValue())
+    $rq: Request(
+             // The assigner must be the current user.
+             assignerId == authenticatedUserId,
+             process == "specific-access",
+             reference != null)
+    $ra: RoleAssignment(
+             status == Status.CREATE_REQUESTED,
+             roleType == RoleType.CASE,
+             roleName == "specific-access-admin",
+             roleCategory == RoleCategory.ADMIN,
+             grantType == GrantType.SPECIFIC,
+             notes != null && notes.size > 0 && notes.toString().length() > 5, /* Make sure there is at least one note with content in it */
+             attributes["caseId"] != null && $caseId : attributes["caseId"].asText(),
+             attributes["jurisdiction"] != null && attributes["jurisdiction"].asText() in ("IA"))
+    $c:  Case(
+             id == $caseId)
+    $e: ExistingRoleAssignment(
+             actorId == $rq.assignerId,
+             roleType == RoleType.ORGANISATION,
+             roleName == "case-allocator",
+             grantType == GrantType.STANDARD,
+             classification.isAtLeast($c.securityClassification),
+             endTime == null || endTime.isAfter(ZonedDateTime.now()),
+             beginTime == null || !beginTime.isAfter(ZonedDateTime.now()),
+             attributes["jurisdiction"] != null && attributes["jurisdiction"].asText() == $c.jurisdiction,
+             attributes["caseType"] == null || attributes["caseType"].asText() == $c.caseTypeId,
+             attributes["region"] == null || attributes["region"].asText() != $c.getRegion(),
+             attributes["baseLocation"] == null || attributes["baseLocation"].asText() != $c.getBaseLocation(),
+             attributes["managedRoleCategory"] == null || attributes["managedRoleCategory"].asText() == $ra.roleCategory.toString(),
+             attributes["managedRole"] == null || attributes["managedRole"].asText() == $ra.roleName)
```

Comment:
The managedRoleCategory and managedRole will never match one to one because it represents a comma separated list. we need to check it with csv contains method already used in case-allocator global rule : ValidationUtil.csvContains($ra.roleCategory.toString(), attributes["allocatedRoleCategory"].asText())
please replace for both the conditions.